pkgs/top-level/python-packages.nix

```diff
@@ -2943,6 +2943,8 @@ self: super: with self; {
     inherit (pkgs) edlib;
   };
 
+  eduvpn-client = callPackage ../development/python-modules/eduvpn-client { };
```

Comment:
This is an end user program so it should not be in pythonPackages.

Reply:
There was some discussion about this in the previous pr: https://github.com/NixOS/nixpkgs/pull/36122#discussion_r172017937. There is a python API, but I'm not sure if it's actually used by anyone as it's not well documented. Should I drop it from pythonPackages nevertheless?